docs/docs/integrating-with-other-libraries.md

```diff
@@ -0,0 +1,270 @@
+
+[Try it on CodePen.](http://codepen.io/wacii/pen/ygzxjG?editors=0010)
+
+## Integrating with Other View Libraries
+
+React can be easily embedded into other applications thanks to the flexibility of `ReactDOM.render()`. While often used once at startup to load a single React application into the DOM, `ReactDOM.render()` can be called multiple times, both to create multiple React applications and to update existing ones. This allows applications to be rewritten in React piece by piece.
+
+### Replacing String Based Rendering with React
+
+A common pattern in older web applications is to describe chunks of the DOM as a string and insert it into the DOM like so: `$el.html(htmlString)`. These points in a codebase are perfect for introducing React. Just rewrite the string based rendering as a React component.
+
+So the following jQuery implementation...
+```js
+const htmlString = '<button id="jquery-btn">jQuery</button>';
+$el.html(htmlString);
+$('jquery-btn').click(() => window.alert('jQuery button'));
+```
+
+...could be rewritten using a React Component.
+```js
+function Button() {
+  return <button id='react-btn'>React</button>;
+}
+ReactDOM.render(
+  <Button />,
+  document.getElementById('react-container'),
+  () => {
+    $('#react-btn').click(() => window.alert('React button'));
+  }
+);
+```
+
+From here you could start moving more logic into the component and begin adopting more common React practices.
+
+```js
+function Button({ handleClick }) {
+  return <button onClick={handleClick}>React</button>;
+}
+ReactDOM.render(
+  <Button handleClick={() => window.alert('React button v2')} />,
+  document.getElementById('react-container')
+);
+```
+
+[Try it on CodePen.](http://codepen.io/wacii/pen/RpvYdj?editors=1010)
+
+### Embedding React in a Backbone View
+
+Backbone Views typically use HTML strings, or string producing template functions, to create the content for their associated DOM element. Like before, this process may be replaced with a React component.
+
+Each view will have an associated component, and when the view renders, `ReactDOM.render()` is used to render the component into the view's `el`.
+
+```js
+function Component({ text }) {
+  return <p>{text}</p>;
+}
+
+const ComponentView = Backbone.View.extend({
+  render() {
+    const text = this.model.get('text');
+    ReactDOM.render(<Component text={text} />, this.el);
+    return this;
+  },
+  remove() {
+    ReactDOM.unmountComponentAtNode(this.el);
+    Backbone.View.prototype.remove.call(this);
+  }
+});
+```
+
+State is maintained in the view as `this.model` and passed to the component as props.
+
+In addition to normal cleanup, event listeners registered through React as well as component state should be removed by calling `ReactDOM.unmountComponentAtNode()`. This is something React normally calls for us, but because we are controlling the application with Backbone, we must do this manually to avoid leaking memory.
```

Comment:
>This is something React normally calls for us

This is ambigious and might give the wrong idea that you don't need to call `ReactDOM.unmountComponentAtNode()` when you use `ReactDOM.render()` in some other contexts (e.g. a purely React app that happens to have those imperative calls for modals or something else).

We should make it clearer that `unmountComponentAtNode()` unmounts the whole React tree from the outside. However *inside* the React tree, React automatically takes care of unmounting individual components when their parents no longer render them. Not sure how to phrase it without going too much into details.